src/Tribe/RSVP.php

```diff
@@ -2302,17 +2695,20 @@ public function parse_attendee_details() {
 	 *
 	 * @since 4.7
 	 *
-	 * @param int $product_id A product post ID
+	 * @param int $ticket_id The ticket ID.
 	 *
-	 * @return int Either the requested quantity of tickets for the product or `0` in
-	 *             any other case.
+	 * @return int Either the requested quantity of tickets or `0` in any other case.
 	 */
-	public function parse_ticket_quantity( $product_id ) {
-		if ( empty( $_POST[ "quantity_{$product_id}" ] ) ) {
-			return 0;
+	public function parse_ticket_quantity( $ticket_id ) {
+		$quantity = 0;
+
+		if ( isset( $_POST['tribe_tickets'][ $ticket_id ]['quantity'] ) ) {
+			$quantity = absint( $_POST['tribe_tickets'][ $ticket_id ]['quantity'] );
+		} elseif ( isset( $_POST["quantity_{$ticket_id}"] ) ) {
+			$quantity = absint( $_POST["quantity_{$ticket_id}"] );
```

Comment:
* :no_entry_sign: Processing form data without nonce verification.
* :no_entry_sign: Array keys must be surrounded by spaces unless they contain a string or an integer.
* :no_entry_sign: There must be a single space after the first square bracket
* :no_entry_sign: There must be a single space before the last square bracket

<sub>hash: 727e2c615984be6b542f59e25bd1e6fa</sub>